dg/lib/error.py

```diff
@@ -42,6 +50,17 @@ def get_parsed_error_message(cls, error_message: str) -> str:
 
         return output
 
+    @classmethod
+    def get_parsed_error_message_without_incident_id(cls, error_message: str) -> str:
+        search_result = regex.search("FAILED\\n(.*)\\n\\n(Incident ID: .*)\\n", error_message, regex.DOTALL)
```

Comment:
Should we add some comment that `regex.DOTALL` allows multiline error messages (or actually allows `dot` to be `new line charachter`). I personally did not know that and had to google it.